docs/setup/settings.asciidoc

```diff
@@ -152,6 +152,9 @@ This value must be a positive integer. *Default: `30000`*
  | Time in milliseconds for {es} to wait for responses from shards.
 Set to 0 to disable. *Default: `30000`*
 
+| `elasticsearch.compression:`
```

Comment:
Since you document it, we need to change the `release_notes` label. Maybe to `enhancements`?